Tasks/ShellScript/task.json

```diff
@@ -53,6 +53,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "disableAutoCwd",
+            "type": "boolean",
+            "label": "Disable Auto CWD",
+            "defaultValue": "false",
```

Comment:
Good suggestion ... But, I'm trying to decide whether cwd should be promoted out of advanced and if this should be left in advanced.  In that case we wouldn't want conditioned.